bin/dev.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node_modules/.bin/tsx
+
+import { execute } from '@oclif/core';
+
+import { error } from '../src/lib/outputs.ts';
```

Comment:
It is on purpose, when running the file directly it will invoke tsx, and there is no `src` file with a `.js` extension

Reply:
so its a `tsx` (or better say `esbuild`) thing? damn why on earth is everything going against each other, this works in a completely opposite way as node, right? :D

Reply:
No, it's correct... Since this script is ran through tsx, it's like running `src/index.ts` in crawlee (only with a `.js` extension)... It probably doesn't work because the dev script is a `.js` file instead of a `.ts` file